lib/rack/attack/cache.rb

```diff
@@ -19,10 +19,10 @@ def initialize(store: self.class.default_store)
 
       attr_reader :store
 
-      def store=(store)
+      def store=(store, bypass_all_store_errors: false, bypassable_store_errors: [])
```

Comment:
How would this be called from the caller? 
Thinking about what the end user API would be.